src/RouteList.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const Ioc = require('adonis-fold').Ioc
+const Command = Ioc.use('Adonis/Src/Command')
+const Route = Ioc.use('Adonis/Src/Route')
+
+class RouteList extends Command {
+
+  constructor (Helpers) {
+    super()
+    this.helpers = Helpers
+  }
+
+  get signature () {
+    return 'route:list'
+  }
+
+  get description () {
+    return 'List all Routes registered for this app'
+  }
+
+  setup(){
+      //Require the app/Http/routes.js to know what routes are registered
+      require(this.helpers.appPath()+'/Http/routes.js')
```

Comment:
Also it will be nice to use the `path` module for building the path, since windows and unix directory `/` are different.